Makefile

```
@@ -3,9 +3,13 @@ OUT := $(shell pwd)
 .PHONY: default
 default: build test
 
-export TAG ?= latest
 export HUB ?= quay.io/maistra-dev
 export ISTIO_VERSION ?= 1.23.0
+export BUILD_TEST_IMAGE ?= true
+
+ifeq ($(BUILD_TEST_IMAGE),true)
+	export TAG := test
+endif
```

I know that the `latest` fallback is used in the test file if not set in environment, but do you think it might be easier for future readers to do something like this?
```suggestion
ifeq ($(BUILD_TEST_IMAGE),true)
	export TAG := test
else
    export TAG ?= latest
endif
```

---

I realized that overwriting `TAG` globally depending on `BUILD_TEST_IMAGE` was a breaking change. Please look once again.